### ISSUE TITLE
Primary Palette just stops working after a while

### ISSUE THREAD

**Issue by ArtyPenguin** [haven1433/HexManiacAdvance#236]
So, there's this bug which I don't know how it happens, but after editing the Primary Palette for a while, at some point, you'll loose access to editing it, this happens a lot and it's not really a good thing, I had it happen on 4 or 5 roms already. It only happens if you do edits to the Palette first as far as I'm concerned, but I'm still not aware as to what exactly causes it and it seems to be random(?). I am using the most recent version. Image for reference, Although I doubt it will be useful. 
![bug](https://github.com/user-attachments/assets/1e822cfe-8425-41ed-85ac-f0f17706e33d)

**Comment by haven1433** [comment]
> random

Sorry, but without exact steps to reproduce the issue, I'd have a hard time tracking down what's going on

**Comment by ArtyPenguin** [comment]
That's mostly the problem, there isn't a consistent step by step method to trigger this. All I'm really sure about is that it happens whenever you edit the Primary Palette a lot of times, It could also be an issue within the actual rom that makes the main Palette unusable, I just got no clue how it happens exactly, I did found a way to solve it, but it's not intuitive at all. You just need to repoint the blockset into a new one, and now, both the new and old blockset will be editable again, so it also could be something related to the actual palette. Sorry if this didn't helped.
